akka-bench-jmh/src/main/scala/akka/http/HttpRequestParsingBenchmark.scala

```diff
@@ -30,7 +30,7 @@ class HttpRequestParsingBenchmark {
   implicit val system: ActorSystem = ActorSystem("HttpRequestParsingBenchmark")
   implicit val materializer = ActorMaterializer()
   val parserSettings = ParserSettings(system)
-  val parser = new HttpRequestParser(parserSettings, false, HttpHeaderParser(parserSettings)())
+  val parser = new HttpRequestParser(parserSettings, false, HttpHeaderParser(parserSettings, NoLogging)())
```

Comment:
Hah, ofc. thanks for the help @Hawstein.
Rushed fixes are the worst fixes, my bad.